pages/1.10/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -8,26 +8,306 @@ excerpt:
 
-1. Using a [custom setup file and the CLI or advanced installer](/1.10/installing), provide the paths to the custom CA certificate, corresponding private key, and optionally corresponding intermediate CA certificates and/or a root CA certificate as part of the chain. All files must be placed in `genconf` directory in the home directory of your bootstrap node. The installer verifies the data.
-1. Before installing DC/OS to all nodes, place the private key file corresponding to the custom CA certificate specified in `ca_certificate_key_path` securely on all master nodes in `/var/lib/dcos/pki/tls/CA/private/custom_ca.key`. The file must be readable by the root user and have 0600 permissions set.
+- **Configuration:** The set of the configuration parameters that governs the specific aspects of the installation procedure. The configuration is stored in the DC/OS configuration file.
+
+- **DC/OS configuration file:** The file which contains the DC/OS configuration parameters. The DC/OS configuration file is normally called `config.yaml` and must be present in the `genconf/` directory on the bootstrap node during the installation. It is used by the DC/OS installer.
+
+- **Installation directory:** The directory on the bootstrap node where the DC/OS installer resides. It is denoted with `$DCOS_INSTALL_DIR` in this document.
+
+# Requirements
+
+In order to install DC/OS Enterprise with a custom CA certificate you will need:
+
+- to use the **Advanced installation method:**  [advanced DC/OS installation method](/1.10/installing/custom/advanced/). Other installation methods are not supported.
+- **A file containing the custom CA certificate**
+- **A file containing the private key associated with the custom CA certificate**
+- If the CA is **not** a root CA, **a file containing the certificate chain associated with the custom CA certificate** 
+
+## Manually placing the custom CA certificate, the associated private key and the certificate chain onto the bootstrap node
+The custom CA certificate, the associated private key and the certificate chain files must be put in the `$DCOS_INSTALL_DIR/genconf/` directory on the bootstrap node:
+
+```bash
+cd $DCOS_INSTALL_DIR
+ls genconf/
+```
+```bash 
+dcos-ca-certificate.crt
+dcos-ca-certificate-key.key
+dcos-ca-certificate-chain.crt
+```
+
+## Manually placing the private key associated with the custom CA certificate onto the master nodes
+
+For security reasons, the installer will not copy the private key from the bootstrap node to the master nodes. 
+The private key associated with the custom CA certificate must be distributed manually to every DC/OS master node **before starting the installation**. The filesystem path for the private key file must be `/var/lib/dcos/pki/tls/CA/private/custom_ca.key`. The directory `/var/lib/dcos/pki/tls/CA/private` should be created manually with the following  command before putting the file `custom_ca.key` in the directory on every DC/OS master node:
```

Comment:
In the last sentence, I think "should" should be replaced with "can". This provides a simple way to perform the steps, but makes it clear that the installer can choose to achieve the same effect in other ways (e.g. using a configuration management tool).

Note, with Markdown, it is helpful when reviewing and diff'ing if each sentence is placed on a separate line. As long as there are no blank lines, the sentences will remain in the same paragraph when rendered.